src/plugins/saved_objects_management/server/routes/find.test.ts

```diff
@@ -0,0 +1,154 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { registerFindRoute } from './find';
+import { ISavedObjectsManagement } from '../services';
+import { IRouter } from 'src/core/server';
+import { managementMock } from '../services/management.mock';
+
+describe('Find route', () => {
+  let router: jest.Mocked<IRouter>;
+  let managementService: jest.Mocked<ISavedObjectsManagement>;
+  let context: any;
+  let req: any;
+  let res: any;
+
+  beforeAll(() => {
+    router = {
+      get: jest.fn(),
+      handleLegacyErrors: jest.fn((handler) => handler),
+    } as any;
```

Comment:
We can use the mock from http service as well I guess and we do not need a type casting here.

Reply:
Done, previously got some issue on import module but now solved